src/app/components/Header.jsx

```diff
@@ -20,9 +20,9 @@ export default function Header() {
 
         {/* Contact Me Button */}
         <Link href="/contact">
-          <button className="bg-gray-800 hover:bg-gray-700 text-white py-2 px-4 rounded-full">
+          <Button className="bg-gray-800 hover:bg-gray-700 text-white py-2 px-4 rounded-full">
```

Comment:
some of these classes are already applied to Button component, please clean them up